app/controllers/api/v1/stats/statistics_controller.rb

```diff
@@ -1,4 +1,5 @@
-class Api::StatsController < Api::BaseController
+class Api::V1::Stats::StatisticsController < Api::BaseController
```

Comment:
Would you rename to `Api::V1::Stats::PublishersController`?

Reply:
We should keep this as statistics for now as there are sections that use only `channels` and not `publishers`. 

ie, `youtube_channels_by_view_count`, `twitch_channels_by_view_count`

I would be willing to do a refactor later on when we add more stats and some patterns emerge

Reply:
I'm okay with leaving `youtube_channels_by_view_count` and moving it to `channels_controller`. `Stats::StatisticsController` sounds redundant, rather `Stats::MiscellaneousController` is more descriptive.